_posts/2022-08-14-data-pipelines-production-ready-great-expectations.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Make Your Trino Data Pipelines Production Ready Using Great Expectations"
+author: "Brian Olsen, Brian Zhan"
+excerpt_separator: <!--more-->
+---
+
+Great Expectations is an open source Python library that allows you to write
+detailed tests called
+[expectations](https://docs.greatexpectations.io/docs/terms/expectation/)
+against your data. Users write these expectations to run validations against the
+data as it enters your system. These expectations are expressed as methods in
+Python and stored in JSON and YAML files. One great advantage of expectations is
+the human readable documentation that results from these tests. As you role out
+different versions of the code, you get alerted to any unexpected changes and 
+have version specific generated documentation for what changed. In this blog, 
+we'll show you how to write expectations on tables in Trino!
+
+![](/assets/blog/data-pipelines-production-ready-great-expectations/trino-ge.png)
+
+
+<!--more-->
+
+## The need for data quality
+
+Managing data pipelines is not for the faint of heart. Nodes fail, you run
+out of memory, bursty traffic causes abnormal behavior, and that’s just the tip
+of the Apache Iceberg 😉. Lots of Trino community members build sophisticated
+data pipelines and data applications using Trino. This received a turbo-boost 
```

Comment:
It's a little unclear what 'this' is referring to when you say "This received," I'd reword.